### DISCUSSION TITLE
[IntelliSense] Linting not working

### DISCUSSION THREAD

**Discussion by whaaaley** [tailwindlabs/tailwindcss#8959] [Help]
**What version of Tailwind CSS IntelliSense are you using?**
v0.8.6

**What version of Tailwind CSS are you using?**
v3.1.6

**What package manager are you using?**
npm

**What operating system are you using?**
debian via wsl on win11

**Reproduction URL**

**Describe your issue**
I'm not able to get linting working on any file types. CSS, SCSS, HTML, Vue files, anything... There's no info in the output either.

tailwind config
``` 
module.exports = {
  content: [
    './index.html',
    './src/**/*.{css,scss,js,ts,jsx,tsx,vue}'
  ],
  theme: {
    screens: {
      sm: '480px',
      md: '768px',
      lg: '960px',
      xl: '1200px'
    }
  }
}
```

vscode settings:
```
{
  "files.associations": {
    "*.css": "tailwindcss",
    "*.scss": "tailwindcss"
  },
  "editor.quickSuggestions": {
    "strings": true
  },
  "tailwindCSS.codeActions": true,
  "tailwindCSS.validate": true
}
```

**Comment by bradlc** [comment]
Hey @whaaaley. Are other features working, such as completions? Do you have an example of some code where you would expect a warning but aren't getting one?

**Reply by whaaaley** [reply]
Thanks for the reply! Yup, auto completions work it was just not linting. I did mange to solve my problem eventually though but not via the plugin. Instead of using the VSCode plugin I setup the prettier plugin through my existing Stylelint configuration in the project.

**Reply by whaaaley** [reply]
Some info about my system if it helps. I'm running vscode 1.69.2 on win11. I don't have any code actions settings set in my settings.json. Only the tailwind options listed above. I don't have a code snippet on hand atm but literally any @apply with any classes wouldn't sort in any combination.

**Reply by whaaaley** [reply]
I did notice just now that tailwindCSS is not all lowercase. I copied this directly out of the documentation so if that's the problem then maybe the docs should be updated.

**Reply by madiha-right** [reply]
Same here. Other features are working well but linting. Any Suggestion?

**Reply by whaaaley** [reply]
I just stopped relying on the plugin. If you have a build toolchain I'd recommend just setting up linting with JS APIs instead. It's much more stable and also version controlled which is much nicer.
